Guides/Anomalies/Anomalies & You - Anomaly Types

```diff
@@ -0,0 +1,50 @@
+Containment Tip: Use fire-resistant equipment and try to isolate these anomalies away from flammable materials. Once scanned, try to contain with Reinforced Plasma Glass Windows whilst also contacting an Atmos Tech to space the floor below the Anom', as the temprature created within the containment will be high enough to kill, once it is broken.
+
+[head=3]Ice Anomalies[/head]
+[bold]Properties:[/bold] Emit intense cold, capable of harming nearby personnel. The also shoot out ice shards which will often break containment rapidly once over 60% Severity. They can spread ice on floors, creating slip hazards.
+[bold]Containment Tip:[/bold] Treat them a lot like the Pyroclastic Anom', though spacing the tile undearneath is optional but recommended, these can be contained with reinforced glass but once over 50-60% severity, use Reinforced Plasma Glass Windows and check on the windows damage regularly.  Wear thermal protection when interacting with ice anomalies, AND BRING A WELDING TOOL.
+
+[head=3]Electric Anomalies[/head]
+[bold]Properties:[/bold] Generate high-voltage discharges, creating electrical surges that can damage electronics and injure personnel. They can short out power grids in their vicinity.
+[bold]Containment Tip:[/bold] Ground the containment area using a grounding rod. These anomalies are best contained in shielded areas away from critical electronics. Relocating the Anom with a Gauntlet is an option as well.
+
+[head=3]Gravitational Anomalies[/head]
+[bold]Properties:[/bold] Warp gravitational fields, causing objects and people to be pulled toward or pushed away from them. They can make moving around difficult and sometimes create dangerous high-gravity zones. Be warned, radiation is also emitted from these anom's.
+[bold]Containment Tip:[/bold] Wear a radiation suit, use lockers or other dragable objects like your A.P.E. to nudge the Anomaly into a better position, I would suggest using Plasma Windows (small/thin) to contain the anom and stop the radiation from causing harm.
+
+[head=3]Flesh Anomalies[/head]
+[bold]Properties:[/bold] Composed of living, organic material, flesh anomalies may spawn hostile creatures if left unchecked. They also emit a foul odor and can spread, affecting their surroundings.
+[bold]Containment Tip:[/bold] Quarantine flesh anomalies and have weapons ready in case of hostile entity emergence. Consider immediate disposal if containment is compromised.
+
+[head=3]Bluespace Anomalies[/head]
+[bold]Properties:[/bold] Bluespace anomalies can warp space, leading to localized teleportation effects. They may cause items or personnel to randomly teleport or phase in and out of the area.
+[bold]Containment Tip:[/bold] Avoid placing unanchored valuable or sensitive equipment nearby, as it may be affected by teleportation. Try to Avoid keeping this anom near the edges of the station and/or public hall ways. Use the APE to stabilize or neutralize quickly.
```

Comment:
Minor spelling
```suggestion
[bold]Containment Tip:[/bold] Avoid placing unanchored valuable or sensitive equipment nearby, as it may be affected by teleportation. Try to Avoid keeping this anom near the edges of the station and/or public hallways. Use the APE to stabilize or neutralize quickly.
```